src/OpenLoco/GameCommands/UpdateOwnerStatus.cpp

```diff
@@ -0,0 +1,29 @@
+#include "../CompanyManager.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x004383CA
+    static uint32_t updateOwnerStatus(const uint8_t flags, OwnerStatus ownerStatus)
+    {
+        if (flags & Flags::apply)
+        {
+            auto companyId = CompanyManager::getUpdatingCompanyId();
+            auto company = CompanyManager::get(companyId);
```

Comment:
can you do `auto*` just to make it explicit that this is a pointer.

Reply:
Done.